cluster/Vagrantfile

```diff
@@ -75,36 +92,65 @@ $1
 EOF
 SCRIPT
 
-# begin execution here
+# Begin execution here
 num_nodes = ENV['CONTIV_NODES'] ? ENV['CONTIV_NODES'].to_i : 2
+num_workers = num_nodes - 1
 base_ip = ENV['CONTIV_IP_PREFIX'] || '192.168.2.'
-start = ENV['CONTIV_KUBEADM'] ? 50 : 50 + num_nodes
-name_start = ENV['CONTIV_KUBEADM'] ? 1 : 1 + num_nodes
-node_ips = Array.new(num_nodes) { |n| base_ip + (n + start).to_s }
-node_names = Array.new(num_nodes) { |n| "contiv-node#{n + name_start}" }
+num_orcs = orchestrators.length
+ip = 50
 
-master_ip = create_etc_hosts(num_nodes, base_ip, start)
-create_cfg_info(num_nodes, node_ips)
+node_names = []
+node_ips = []
+node_orcs = []
+master_ips = []
+
+orchestrators.each do |o|
```

Comment:
what does this outer loop do? Please add comments. In particular, why it is dealing with multiple orchestrators? I thought we setup a single cluster at a time.